AnnoyancesFilter/sections/push-notifications.txt

```diff
@@ -21,6 +21,7 @@ macwelt.de,pcwelt.de#%#AG_setConstant('firebase.messaging', 'noopFunc');
 ||pushtora.com^$third-party
 !
 ||list-news.net^$third-party
+||sometired.com^$third-party
```

Comment:
This is fake push notification (you can right click on it and select source), please move to adservers.